BillTrack.Persistence/Configuration/EmployeeConfiguration.cs

```diff
@@ -0,0 +1,58 @@
+using BillTrack.Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace BillTrack.Persistence.Configuration;
+
+public class EmployeeConfiguration : IEntityTypeConfiguration<Employee>
+{
+    public void Configure(EntityTypeBuilder<Employee> builder)
+    {
+        builder.Property(e => e.Email)
+            .HasMaxLength(100)
+            .IsRequired();
+
+        builder.Property(e => e.Firstname)
+            .HasMaxLength(100)
+            .IsRequired();
+
+        builder.Property(e => e.Lastname)
+            .HasMaxLength(100)
+            .IsRequired();
+
+        builder.Property(e => e.Salary)
+            .IsRequired();
+
+        builder.Property(e => e.DepartmentId)
+            .IsRequired();
+
+        builder.Property(e => e.ProjectId)
+            .IsRequired();
+        
+        builder.HasIndex(e => e.DepartmentId)
+            .IsUnique();
+
+        builder.HasIndex(e => e.ProjectId)
+            .IsUnique();
```

Comment:
the same for `ProjectId`. Only 1 employee for a project will be allowed.

Reply:
fixed